src/App.tsx

```diff
@@ -1,25 +1,112 @@
+import { FormEvent, useState } from 'react';
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [prevTodos, setPrevTodos] = useState(todos);
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [completed] = useState(false);
+  const [isTitleCorrect, setIsTitleCorrect] = useState(false);
+  const [isUserIdCorrect, setIsUserIdCorrect] = useState(false);
+
+  const getTodo = () => {
+    return (
+      {
+        id: Math.max(0, ...prevTodos.map(({ id }) => id + 1)),
+        title,
+        userId,
+        completed,
+        user: getUser(userId) || null,
```

Comment:
getUser already returns null
```suggestion
        user: getUser(userId),
```